dist/profile/manifests/catchall.pp

```diff
@@ -69,34 +69,33 @@
 
   # permalinks
   # - this one is referenced from 1.395.1 "sign post" release
-  Redirect /why            https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=53608972
+  # TODO: Create a Jenkins Manifesto page?
+  Redirect /why            https://www.jenkins.io/
   # baked in the help file to create account on Oracle for JDK downloads
   Redirect /oracleAccountSignup    http://www.oracle.com/webapps/redirect/signon?nexturl=http://jenkins-ci.org/
-  # to the donation page
-  Redirect /donate        https://wiki.jenkins-ci.org/display/JENKINS/Donation
   # CLA links used in the CLA forms
-  Redirect /license        https://wiki.jenkins-ci.org/display/JENKINS/Governance+Document#GovernanceDocument-cla
-  Redirect /licenses        https://wiki.jenkins-ci.org/display/JENKINS/Governance+Document#GovernanceDocument-cla
+  Redirect /license        https://www.jenkins.io/project/governance/#cla
+  Redirect /licenses        https://www.jenkins.io/project/governance/#cla
   # used to advertise the project meeting
-  Redirect /meetings/        https://wiki.jenkins-ci.org/display/JENKINS/Governance+Meeting+Agenda
+  Redirect /meetings/        https://www.jenkins.io/project/governance-meeting/
   # used from friends of Jenkins plugin to link to the thank you page
-  Redirect /friend        https://wiki.jenkins-ci.org/display/JENKINS/Donation
+  Redirect /friend        https://www.jenkins.io/donate
```

Comment:
```suggestion
  Redirect /friend        https://www.jenkins.io/donate/
```

Reply:
oh, good catch! the link `https://www.jenkins.io/donate` redirects to `https://wiki.jenkins-ci.org/display/JENKINS/Donation`, while `https://www.jenkins.io/donate/` is ok.

Reply:
Not an actual catch because https://github.com/jenkins-infra/jenkins-infra/pull/1907/files#diff-2b2dd788d03cea552c387130532e5f5b364b5617421e6c240f0c2fbdb45f3fbaL75-L76 would take care of that. This just saves an extra redirect.